addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/CrossDomainScriptInclusionScanRuleUnitTest.java

```diff
@@ -157,13 +160,12 @@ void crossDomainScriptWithIntegrity() throws HttpMalformedHeaderException {
 
         HttpMessage msg = new HttpMessage();
         msg.setRequestHeader("GET https://www.example.com/test/ HTTP/1.1");
-
         msg.setResponseBody(
                 "<html>"
                         + "<head>"
                         + "<script src=\"https://www.example.com/script1\"/>"
                         + "<script src=\"https://www.otherDomain.com/script2\" integrity=\"12345678\"/>"
-                        + "<head>"
+                        + "</head>"
```

Comment:
Nice catch, it should be fixed in all tests though.

Reply:
Done, but it was not the issue.

Reply:
Right, Jericho is able to parse that fine.